### PR TITLE
README.md for use by GitHub

### PR DESCRIPTION
**Pull request by andyjefferson**
This provides a README.md file using the contents of the README.html as basis, with a few minor adjustments in formatting etc.

Since I don't have commit access to "db-jdo" despite having my GitHub id associated to my Apache "id" (never mind all of my old commits using SVN not being tagged against this github account) somebody else will have to commit this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,12 +120,12 @@ Example 2 : Runs the test configurations specified in `alltests.conf` and `cfg1.
...
-* * * jdo.tck.testdata : The fully qualified file name of the xml test data file(optional)
-* * * jdo.tck.standarddata : The fully qualified file name of the xml test data file(optional)
-* * * jdo.tck.mapping : The file designator that maven.xml uses to build a javax.jdo.option.Mapping value and corresponding schema name (required)
+    * test configuration files (*.conf). Each of these files sets values for
+        * jdo.tck.testdescription : An optional string describing the purpose of these tests
+        * jdo.tck.classes : A list of one or more test classes (required)
+        * jdo.tck.testdata : The fully qualified file name of the xml test data file(optional)
+        * jdo.tck.standarddata : The fully qualified file name of the xml test data file(optional)
+        * jdo.tck.mapping : The file designator that maven.xml uses to build a javax.jdo.option.Mapping value and corresponding schema name (required)
 * exclude.list  : A list of test classes NOT to execute during a TCK test run</LI>
```
Comment (clr-apache) [inline README.md]: the trailing </LI> seems to be out of place

Comment (andyjefferson) [reply README.md]: updated in latest commit

**Comment by andyjefferson** [comment]
FWIW You can view the GitHub display of this markdown at https://github.com/andyjefferson/db-jdo/blob/master/README.md

**Comment by clr-apache** [comment]
Just two final comments.

This sentence unfortunately line splits between the "-" and "D". Is there some way to avoid it splitting there? "As a workaround you may set the system property by adding -Duser.home=%HOME% to the environment variable MAVEN_OPTS."
The instructions are on the github page but they describe how to check out the project via svn. Have you considered updating the instructions to use the git repo (either/or gitbox or github)?

**Pull request merged**